impact.tex

```diff
@@ -57,7 +54,24 @@ \subsection{Pathway toward impact}
 \end{draft}
 
 
-\TOWRITE{}{Start with brief narrative summary, hitting 'Scale and Significance' hard. We have close to nothing on Significance, yet. Scale could be clearer: All researchers!}
+Almost all researchers perform some kind of computation in the course of their work,
+from large-scale simulations to production of simple charts and figures.
+Researchers today are faced with a great variety of useful
+and increasingly open tools to perform their research tasks.
+However, this variety presents its own challenge.
+Because we are building and extending \textbf{generic tools for computational reproducibility}.
+We are not solving the \emph{whole reproducibility problem},
+but we \emph{are} solving a big piece of the problem (\textbf{reproducible computational environments}) in a way that can be reused,
```

Comment:
It is quite "negative" sentence. Why do you need to stress so much that you do not solve the whole reproducibility problem?

I think if you say you are solving a big piece it already means you don't solve everything.

Reply:
I'm apparently allergic to boasting language, thanks for pointing that out. I'll try to be stronger.

Reply:
your competitors are not...